src/schemas/json/smapi-manifest-3.17.0.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$comment": "https://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest",
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "semver": {
```

Comment:
Optional fix: move this definition to `base.json` schema to make it available to all schemas at once.